routeros/resource_interface_bridge.go

```diff
@@ -37,6 +37,7 @@ func ResourceInterfaceBridge() *schema.Resource {
 		"auto_mac": {
 			Type:     schema.TypeBool,
 			Optional: true,
+			Default:  false,
```

Comment:
This is proper provider behavior. The `true` value is returned exactly according to the documentation: https://wiki.mikrotik.com/wiki/Manual:Interface/Bridge.
`auto-mac (yes | no; Default: *yes*)`
So we need to revert this fix back :)

Reply:
Sorry just saw your comment, I thought I'd already removed that early but missed it in commit - just removed that one too a little while ago. Cheers.